USERGUIDE.md

```diff
@@ -1148,6 +1148,23 @@ extension ViewController: MiniAppMessageDelegate {
 }
 ```
 
+<a id="miniapp-sendAnalytics"></a>
+
+### Receive MiniApp Analytics 
+
+MiniApps can send the analytics information to host app through the `sendAnalytics` interface defined in JS-SDK.
+iOS Hostapp can extend this optional `MAAnalayticsDelegate` interface/delegate to receive the analytics that is sent from the Miniapp.
+
+```swift
+extension ViewController: MiniAppMessageDelegate {
+    func didReceiveMAAnalytics(analyticsInfo: MAAnalytics, completionHandler: @escaping (Result<MASDKProtocolResponse, MAAnalyticsError>) -> Void) {
```

Comment:
```suggestion
    func didReceiveMAAnalytics(analyticsInfo: MAAnalyticsInfo, completionHandler: @escaping (Result<MASDKProtocolResponse, MAAnalyticsError>) -> Void) {
```

Reply:
Thank you, I have updated this 👍🏽 please review.